### ISSUE TITLE
Declare 'react' as a depdency in `embla-carousel` package

### ISSUE THREAD

**Issue by chitchu** [davidcetinkaya/embla-carousel#212]
### Feature request is related to

- [x] The React version

### Is your feature request related to an issue?

We can't use the package in a yarn pnp enabled project. 

The resolver throws the following error:

```
embla-carousel tried to access react, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.
```

### Describe the solution you'd like

Updating the `embla-carousel/package.json` to move "[react](https://github.com/davidcetinkaya/embla-carousel/blob/master/packages/embla-carousel/package.json#L65)" from `devDependencies` into `dependencies`

**Comment by davidcetinkaya** [comment]
Hi Vicente (@chitchu),

Thank you for creating this issue. I don't think the `react` package should be a **dependency** of the `embla-carousel` package?  It should ideally be a **peerDependency**, because `embla-carousel` requires you to install `react` on your own. Embla shouldn't include a duplicate `react` in the consumer project because this is known to cause a lot of problems. The reason why it's not a **peerDependency** today has been discussed in the following issue #92.

In the future, I'm hoping to separate the react version to a separate package called `embla-carousel-react`, but I haven't had the time to do so. Doing that will make it possible to move `react` to **peerDependencies**.

Best,
David

**Comment by chitchu** [comment]
Ok thanks for clarifying. Looking forward to the dedicated react version.

**Comment by davidcetinkaya** [comment]
Hi Vicente (@chitchu),

Until then, if you find a solution to the yarn error, please post it here. It might help other.

Cheers,
David

**Comment by davidcetinkaya** [comment]
@chitchu the **dedicated react version** is on its way here: #220.

**Comment by davidcetinkaya** [comment]
Hi @chitchu,

I've just released v5.0.0 and the React version is now a [dedicated React package](https://www.npmjs.com/package/embla-carousel-react). I hope that this will solve the issues you were experiencing.

Cheers,
David
